app/routes/api/campaign.js

```diff
@@ -69,6 +69,26 @@ router.route('/campaigns/:campaignId')
         .catch(function(err) {
             response.json(err);
         });
+})
+
+.delete((request, response) => {
```

Comment:
Moved the campaign delete logic from being a full page URL to an API route instead so that the page doesn't need to refresh after deleting a campaign.